docs/user-guide/admin-cli.md

```diff
@@ -317,22 +317,20 @@ Remove a tenant, optionally all volumes for a tenant can be removed as well.
 
 Sample:
 ```
-/usr/lib/vmware/vmdkops/bin/vmdkops_admin.py tenant rm tenant1 --remove-volumes True
+/usr/lib/vmware/vmdkops/bin/vmdkops_admin.py tenant rm --name=tenant1 --remove-volumes
```

Comment:
From a usability stand point 
```
vmdkops_admin tenant create T1
vmdkops_admin tenant rm T1
``` 
is a lot nicer, since all the operations are done on a per tenant basis maybe we should change all
```
vmdkops_admin tenant T1 access add --rights all
```